filament/src/PostProcessManager.cpp

```diff
@@ -1308,27 +1308,50 @@ FrameGraphId<FrameGraphTexture> PostProcessManager::bloomPass(FrameGraph& fg,
         });
     }
 
+    // TODO: this is a feature flag that should be passed in from above.
```

Comment:
I would maybe add a comment here explaining what "feedback" means and why it's necessary to disable it in certain cases.

Reply:
Good catch, this term is from the GL spec, I amended the comment by fleshing out this comment block.